src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/BitbucketApiFactory.java

```diff
@@ -28,14 +27,14 @@
      * repository.
      *
      * @param serverUrl   the server URL.
-     * @param credentials the (optional) credentials.
+     * @param authenticator the (optional) authenticator.
      * @param owner       the owner name.
      * @param repository  the (optional) repository name.
      * @return the {@link BitbucketApi}.
      */
     @NonNull
     protected abstract BitbucketApi create(@Nullable String serverUrl,
-                                           @Nullable StandardUsernamePasswordCredentials credentials,
+                                           @Nullable BitbucketAuthenticator authenticator,
```

Comment:
This is a protected method with a changed signature. Is that a concern for API compatibility? In theory there could be other sub-classes outside the ones this plugin provides.

Reply:
Someone might have written an extension, please consider adding the ability to accept StandardUsernamePasswordCredentials and convert them to BitbucketAuthenticator

Reply:
Nvm already done 😇